app/models/reservation.rb

```diff
@@ -118,9 +120,15 @@ def self.relay_in_progress
     where("actual_start_at IS NOT NULL AND actual_end_at IS NULL")
   end
 
+  scope :not_offline, -> { where(type: nil) }
```

Comment:
Do we still have `nil` types? I thought the base reservation would be saved as `Reservation`